.circleci/config.yml

```diff
@@ -982,14 +990,18 @@ workflows:
                 - release
                 - master
                 - beta
-      - plugin-amplify_e2e_tests:
+      - customStacks-amplify_e2e_tests:
           filters: &ref_2
             branches:
               only:
                 - master
                 - split-test
           requires:
             - publish_to_local_registry
+      - plugin-amplify_e2e_tests:
```

Comment:
Why do have this? Isn't this unrelated?

Reply:
This is diffing algorithm doing its thing